app/javascript/packs/application.js

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import 'font-awesome/css/font-awesome.css';
 import '../styles/application.scss';
-
+require("babel-polyfill");
```

Comment:
`babel-polyfill` also "`require`" unnecessary things. You should use "useBuiltIns: true" of `babel-preset-env`.